### ISSUE TITLE
JAWS repeats readouts inappropriate number of times

### ISSUE THREAD

**Issue by loganbraywork** [phetsims/john-travoltage#417]
**Test device**

Windows 10 Laptop

**Operating System**

Windows 10

**Browser**

Chrome 90.0.4430.93 & JAWS 2021.210.3174

**Problem description**

For https://github.com/phetsims/QA/issues/645

When using JAWS without voicing enabled, if the leg is moved before JAWS is able to respond the sim will repeat the final readout an incorrect number of times.

**Steps to reproduce**

1. Move leg to the furthest point either to the left or right
2. Move leg to the opposite furthest point on either the left or right
3. Observe

**Visuals**

https://user-images.githubusercontent.com/54685448/118042417-48bc4700-b331-11eb-82e3-0512c325f60e.MOV

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪John Travoltage‬
URL: https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-preferences.4/phet/john-travoltage_en_phet.html
Version: 1.6.0-preferences.4 2021-04-28 22:49:44 UTC
Features missing: applicationcache, generatedcontent, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36
Language: en-US
Window: 1536x754
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

**Comment by jessegreenberg** [comment]
Thanks @loganbraywork. Unfortunately I think this is a bug with JAWS and not the sim. JAWS will do this will all sliders on the web, not just the leg in this sim. I believe this is listed in https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-preferences.6/phet/john-travoltage_en_phet.html, can you please confirm? If it is the same issue I think this can be closed.

**Comment by jessegreenberg** [comment]
I am pretty certain this is the known JAWS behavior problem. Good to know that it still exists but there is nothing for PhET to do about this. Closing.
